CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- The `lti_id` field is optional on Videos and Documents.
```

Comment:
You must move it to `Unreleased` section, a new released exists since you worked on this PR.